device-types/Raritan/PX2-5205R.yaml

```diff
@@ -0,0 +1,39 @@
+---
+manufacturer: Raritan
+model: PX2-5205R
+slug: px2-5205r
```

Comment:
All slugs should have the manufacturer's name appended to them. This should start with `raritan-`